packages/graalvm/src/test/kotlin/elide/runtime/gvm/internals/js/node/NodeBufferTest.kt

```diff
@@ -107,7 +107,23 @@ import elide.testing.annotations.TestCase
     """
   }
 
-  @Test fun `isUtf8() compliance`() = conforms {
+  @Ignore @Test fun `isAscii() compliance (invalid)`() = conforms {
```

Comment:
If this was broken, it was probably related to the missing `Buffer` symbol

Reply:
i thought so too, but alas, it's still broken for some reason

Reply:
this one _did_ fix, so did `valid` (for `isAscii`)